src/cpu/x64/gemm/f32/jit_avx512_common_gemm_f32.cpp

```diff
@@ -1766,8 +1767,7 @@ dnnl_status_t jit_avx512_common_gemm_f32(int nthrs, const char *transa,
         ompstatus = (unsigned char volatile *)ompstatus_;
         assert(ompstatus);
 
-        for (int i = 0; i < nthr_to_use; i++)
-            ompstatus[i * CACHE_LINE_SIZE] = 0;
+        std::fill_n(ompstatus, nthr_to_use * CACHE_LINE_SIZE, 0);
```

Comment:
I still think this one is totally wrong. See how the original skips by cache line size? Can't be a fill.